src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -371,3 +373,28 @@ def validate_snapshot_import(namespace):
             raise InvalidArgumentValueError("--src-snapshot is only applicable when importing from a configuration store.")
         if any([namespace.src_key, namespace.src_label, namespace.skip_features]):
             raise MutuallyExclusiveArgumentError("'--src-snapshot' cannot be specified with '--src-key', '--src-label', or '--skip-features' arguments.")
+
+
+def validate_sku(namespace):
+    if namespace.sku.lower() == 'free':
```

Comment:
If we do not specify a `sku` when creating a store, could be get a null ref here? I know the sku value defaults to standard but I want to confirm that the default is already passed to the namespace.

Reply:
Thats a good point. I tried it out it seems to default to Standard and already passed to the namespace before validation. I am getting no error.